packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -210,12 +210,13 @@ export class DefaultCrudRepository<T extends Entity, ID>
   }
 
   async findById(id: ID, filter?: Filter, options?: Options): Promise<T> {
+    options = options || {};
+    // WIP: introducing `strictFind` to return `404` for non-existent resources. Currenyly returns `null`
+    options.strictFind = true;
```

Comment:
Should this be configurable at the `@model()` level similar to `strict` and `strictDelete`?